src/applications/static-pages/ask-va/components/App/index.unit.spec.js

```diff
@@ -0,0 +1,63 @@
+// Node modules.
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+// Relative imports.
+import environment from 'platform/utilities/environment';
+import { App } from '.';
+
+describe('Ask VA <App>', () => {
+  it('does not render when explicitly hidden', () => {
+    const wrapper = shallow(<App hidden />);
+    expect(wrapper.text()).to.equal('');
+    wrapper.unmount();
+  });
+
+  it('renders what we expect when not authenticated', () => {
+    const expectedHref = environment.isProduction()
+      ? 'https://ava.va.gov/'
+      : 'https://ask-staging.va.gov';
+    const wrapper = shallow(<App loa={undefined} />);
+    expect(wrapper.find(`a[href="${expectedHref}"]`)).be.have.lengthOf(1);
+    expect(wrapper.find(`a[href="${expectedHref}"]`).text()).to.equal(
+      'Ask us a question online',
+    );
+    wrapper.unmount();
+  });
+
+  it('renders what we expect when user is LOA1', () => {
+    const expectedHref = environment.isProduction()
+      ? 'https://ava.va.gov/Unauthenticated'
```

Comment:
If they're authenticated we direct them to an unauthenticated page?

Reply:
If they are LOA1, correct 🙂

Reply:
For context: https://github.com/department-of-veterans-affairs/va.gov-team/issues/28848#issuecomment-907259727

Reply:
![image](https://user-images.githubusercontent.com/12773166/133282208-dee3a9bf-3a46-4278-ab98-e448209f7156.png)

Reply:
I'm assuming it's this way due to LOA1 users not having necessary credentials (first name, last name, email, etc.) for a personalized ask VA experience, but just a guess 🙃